src/execution_plan/ops/op.h

```diff
@@ -17,51 +17,63 @@
 #define OP_REQUIRE_NEW_DATA(opRes) (opRes & (OP_DEPLETED | OP_REFRESH)) > 0
 
 typedef enum {
-	OPType_AGGREGATE,
```

Comment:
Why this reshuffling ?

Reply:
Not strictly necessary, but I wanted to group ops by function more logically. Especially now that we have types like `OPType_NODE_BY_ID_SEEK` and `OpType_NODE_BY_LABEL_AND_ID_SCAN`, it's easy for some to get lost.